test/e2e/metrics_test.go

```diff
@@ -216,35 +217,16 @@ func (tc *testContext) testWindowsPrometheusRules(t *testing.T) {
 
 // getPrometheusToken returns authorization token required to access Prometheus server
 func (tc *testContext) getPrometheusToken() (string, error) {
-	// get secrets from monitoring namespace
-	monitoringSecrets, err := tc.client.K8s.CoreV1().Secrets(monitoringNamespace).List(context.TODO(), metav1.ListOptions{})
-	if err != nil {
-		return "", err
-	}
-
-	// access prometheus-k8s-token-* secret
-	var secretName string
-	for _, secret := range monitoringSecrets.Items {
-		if strings.Contains(secret.Name, "prometheus-k8s-token") {
-			secretName = secret.Name
-			break
-		}
+	tokenRequest := &auth.TokenRequest{
+		Spec: auth.TokenRequestSpec{},
 	}
+	token, err := tc.client.K8s.CoreV1().ServiceAccounts(monitoringNamespace).CreateToken(context.TODO(),
```

Comment:
this creates a new token each time it is executed right? do the tokens expire, or do we have to revoke them somehow?

Reply:
> this creates a new token each time it is executed right? 

Yes

Reply:
> Do we have to revoke them somehow?

No.

> Do the tokens expire, 
Yes, the default ExpirationSeconds is 3600 seconds (1 hour)

Reply:
Please add the token expiration time to the function doc